ios/browser/api/favicon/favicon_attributes.mm

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "brave/ios/browser/api/favicon/favicon_attributes.h"
+#include "ios/chrome/common/ui/favicon/favicon_attributes.h"
```

Comment:
I just realized these files are a copy of these Chromium ones and the header doesn't use any C++, why are we bothering to do this instead of just exporting the header directly? We should just export the symbol and copy the header and delete these files

Reply:
Cannot mark a `symbol` exportable without putting `OBJC_EXPORT` on the original `@interface`. I tried that.